packages/apidom-ns-workflows-1/test/refractor/elements/SuccessAction/index.ts

```diff
@@ -0,0 +1,26 @@
+import { expect } from 'chai';
+import { sexprs } from '@swagger-api/apidom-core';
+
+import { SuccessActionElement } from '../../../../src';
+
+describe('refractor', function () {
+  context('elements', function () {
+    context('SuccessActionElement', function () {
+      specify('should refract to semantic ApiDOM tree', function () {
+        const successActionElement = SuccessActionElement.refract({
+          type: 'goto',
+          stepId: 'getPetStep',
```

Comment:
We want to do exhaustive fixed fields test here.
```suggestion
          stepId: 'getPetStep',
          workflowId: 'uniqueId',
```